setup.py

```diff
@@ -4,11 +4,12 @@
 import os
 import sys
 
-if os.environ.get("SENTRY_PYTHON3") == "1" and sys.version_info[:2] != (3, 6):
-    sys.exit("Error: Sentry [In EXPERIMENTAL python 3 mode] requires Python 3.6.")
+version = sys.version_info
+if os.environ.get("SENTRY_PYTHON2") == "1" and version[:2] != (2, 7):
+    sys.exit("Error: Sentry [In DEPRECATED Python 2 mode] requires Python 2.7.")
 
-if os.environ.get("SENTRY_PYTHON3") != "1" and sys.version_info[:2] != (2, 7):
-    sys.exit("Error: Sentry requires Python 2.7.")
+if os.environ.get("SENTRY_PYTHON2") != "1" and version[0] == 3 and version[1] >= 6:
+    sys.exit("Error: Sentry requires at least Python 3.6.")
```

Comment:
@evanh since you added the check, I would like your consent to change the requirement to be at least Python 3.6 instead of exact match.

I want this as it will likely be needed for the Big Sur support.

Reply:
You mean @EvanPurkhiser :)

Reply:
Yeah I'm fine with loosening the constraint, but I would prefer if we could warn if a user was using something else